pages/samvera/developer_resources/collections/hyrax_2.1/nesting-faq.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "FAQ - Collection Nesting"
+keywords: Nesting, Collection, FAQ
+categories: How to Do All the Things
+permalink: collection-nesting-faq.html
+folder: samvera/how-to/collection_extensions/hyrax_2.1/nesting-faq.md
+sidebar: home_sidebar
+tags: [development_resources]
+a-z: ['FAQ - Collection Nesting', 'Collections - Nesting', 'Nesting of Collections']
+version:
+  label: 'targeted for Hyrax v2.1.0'
+  branch:
+    label: 'collections-sprint'
+    link: 'https://github.com/samvera/hyrax/tree/collections-sprint'
+---
+
+### What is collection nesting?
+
+Collections that have collections as members.
+
+### Can a collection have collections and works as members?
+
+Yes.  A collection can have all sub-collections, all works, or a combination of both.
+
+### Is there a limit to how deep the nesting can go?
+
+The depth of nesting is limited. By default the limit is set to 5.  You can modify the limit by...  
+
+<b style='padding-left: 30px; color: purple; font-size: 1.3em;'>TODO: how to change limit?</b>
```

Comment:
changing the limit in the configuration setting (config/initializers/samvera-nesting_indexer_initializer.rb)